### ISSUE TITLE
Add cache clearing option for the JumpMenu

### ISSUE THREAD

**Issue by litzinger** [ExpressionEngine/ExpressionEngine#623]
The `clear_caching` function lumps the Jump Menu clearing in with the `all` option. For backwards compatibly, if an add-on needs to clear the Jump Menu cache, it has to call `ee()->functions->clear_caching('all');`, which may nuke too much of the cache. To make it compatible with EE 5 and 6, and give add-ons the option of clearing just the Jump Menu cache, what if the following change is made?

```
public function clear_caching($which, $sub_dir = '')
{
    $options = array('page', 'db', 'tag', 'sql');

    if (in_array($which, $options))
    {
        ee()->cache->delete('/'.$which.'_cache/');
    }
    elseif ($which === 'jumpmenu')
    {
        ee('CP/JumpMenu')->clearAllCaches();
    }
    elseif ($which === 'all')
    {
        foreach ($options as $option)
        {
            ee()->cache->delete('/'.$option.'_cache/');
        }
        ee('CP/JumpMenu')->clearAllCaches();
    }
}
```

This would make it so an add-on could call `ee()->functions->clear_caching('jumpmenu');` and in EE 5, nothing happens, but in EE 6, _just_ the `jumpmenu` cache folder is deleted.

**Comment by intoeetive** [comment]
@litzinger we'll add this for 6.0.1

**Comment by ops-andy** [comment]
fixed in 6.0.1
